### DISCUSSION TITLE
Change line spaces ?

### DISCUSSION THREAD

**Discussion by octalflow** [litehtml/litehtml#379] [Q&A]
hello.
I want to change the space between lines.

I changed font's tmHeight value but It affected draw_borders.(I don't want it)

How to do it ?

thank you.

**Comment by tordex** [comment]
```line-height``` css property should work.

**Comment by octalflow** [comment]
ah....

sorry but How to use it ?

**Comment by tordex** [comment]
```html
<p style="line-height: 1.5em">Some text</p>
```

**Reply by octalflow** [reply]
If I change whole lines how to do that ?

My app has many lines.
I want to change all line spaces.

**Reply by tordex** [reply]
You can apply line-height style to the ```<body>``` tag. It is inherited property.

**Reply by octalflow** [reply]
ah..
thank you.
